pandablocks/commands.py

```diff
@@ -293,6 +294,30 @@ def execute(self) -> ExchangeGenerator[None]:
         assert ex.line == "OK"
 
 
+@dataclass
+class Append(Command[None]):
+    """Append the value of a table field.
+
+    Args:
+        field: The field, attribute, or star command to append
+        value: The value, list of strings, to append
+
+    For example::
+
+        Put("SEQ1.TABLE", ["1048576", "0", "1000", "1000"])
+    """
+
+    field: str
+    value: Union[List[str]] = ["",]
+
+    def execute(self) -> ExchangeGenerator[None]:
+        if isinstance(self.value, list):
+            # Multiline table with blank line to terminate
+            ex = Exchange([f"{self.field}<<"] + self.value + [""])
+            yield ex
+            assert ex.line == "OK"
```

Comment:
...and then we don't need the isinstance

```suggestion
        # Multiline table with blank line to terminate
        ex = Exchange([f"{self.field}<<"] + self.value + [""])
        yield ex
        assert ex.line == "OK"
```